---
title: BIIF Newsletter
---
![BIIF logo](/images/biif_logo_white.png )

Dear All,

we would like to draw your attention to **two seminars organized by the Center for Image Analysis/CBA** here in Uppsala and online tomorrow, Tuesday, June 13. 
For those in Uppsala: Seminars will take place in the Theatrum Visuale, Ångström house 10.

## AI-driven computational pathology, June 13, 13:15-14:00
Speaker: **Pekka Ruusuvuori**, Bioimage informatics group leader, Inst. of Biomedicine, University of Turku  
Zoom-link [here](https://uu-se.zoom.us/j/62216376960)  
Host: Nataša Sladoje, Uppsala University

Digital pathology is rapidly transforming the workflow in routine diagnostics, enabling the use of computational methods for interpreting the data. Our aim is to enable faster, less subjective and in some cases even more accurate diagnostics through deep learning based computational pathology. Besides enabling decision support for tasks currently done by human experts, computational pathology has the potential for novel discoveries from histopathology beyond the limits of human vision. Our research focus is on data-intensive research questions in cancer research and computational pathology. The presentation will cover some of our recent work in AI-driven cancer diagnostics, reconstruction and visualization of histological data in 3D, and in developing virtual staining for unstained tissue.

## Dimension reduction: in general, and in single cell data analysis, June 13, 14:15-15:00
Speaker: **Fred Hamprecht**, Professor in Image Analysis and Learning, Heidelberg University   
Zoom-link [here](https://uu-se.zoom.us/j/62216376960)  
Host: Nataša Sladoje, Uppsala University

Visualizing high-dimensional data is an essential step in exploratory data analysis, and it has become routine to show UMAP embeddings, including of single cell data. But how canonical is the use of UMAP, and what do we know about its inner workings? The first part of the talk will provide a brief tutorial overview over strategies for dimension reduction, including neighbor embeddings such as UMAP and tSNE, force-directed layout and auto encoders. In a second part, I will highlight our own contributions, including work on the precise relation between t-SNE and UMAP [Neurips 2022, ICLR 2023], tree-biased auto-encoders [Bioinformatics 2021] and geometric auto-encoders [ICML 2023].
Finally, using the example of STAGATE [Dong&Zhang 2022] I will discuss how spatial cues can inform dimension reduction in spatial transcriptomics. Altogether, and especially in spatial omics, the problem of dimension reduction is anything but "solved".

You can find more information about seminars at CBA [here](https://www.it.uu.se/about_us/divisions/vi3/seminars-list/image_analysis)

Best regards  
Anna, in behalf of BIIF
